l10n/en/firefox/faq.ftl

```diff
@@ -0,0 +1,100 @@
+# $url4 (url) - link to https://www.mozilla.org/firefox/linux/
+# $url5 (url) - link to https://www.mozilla.org/firefox/mobile/
+
+you-can-easily = You can easily <a href="{ $url }">download the { -brand-name-firefox } desktop browser</a> here. { -brand-name-firefox } works on <a href="{ $url2 }">{ -brand-name-windows }</a>, <a href="{ $url3 }">{ -brand-name-mac-short }</a> and <a href="{ $url4 }">{ -brand-name-linux}</a> devices, and is also available for <a href="{ $url5 }">{ -brand-name-android } and { -brand-name-ios }</a>. Make sure you’re downloading our browser from one of our trusted { -brand-name-mozilla }/{ -brand-name-firefox } pages.
+
+is-firefox-free = Is { -brand-name-firefox } free?
+yep-the-firefox = Yep! The { -brand-name-firefox-browser } is free. Super free, actually. No hidden costs or anything. You don’t pay anything to use it, and we don’t sell your personal data.
+related-questions-free = Related questions: is { -brand-name-firefox-browser } free, does { -brand-name-firefox } cost money
+
+is-chrome-better = Is { -brand-name-chrome } better than { -brand-name-firefox }?
+no-we-dont = No, we don’t think { -brand-name-chrome } is better than { -brand-name-firefox }, and here is why: when people ask which browser is better, they’re really asking which browser is faster and safer. { -brand-name-firefox } is updated monthly to make sure you have the speediest browser that respects your privacy automatically.
+
+# Variables:
+# $url (url) - link to https://www.mozilla.org/firefox/browsers/compare/chrome/
+
+see-how-firefox = <a href="{ $url }">See how { -brand-name-firefox } compares { -brand-name-chrome }.</a>
+
+related-questions-better = Related questions: is { -brand-name-firefox } better than { -brand-name-chrome }, is { -brand-name-firefox } better than { -brand-name-google }, is { -brand-name-firefox } safer than { -brand-name-chrome }, is { -brand-name-firefox } more private than { -brand-name-chrome }
+is-firefox-safe-download = Is { -brand-name-firefox } safe to download?
+
+# Variables:
+# $url (url) - link to https://www.mozilla.org/firefox/new/
+
+protecting-your-privacy = Protecting your privacy is our number one priority, and we ensure that installing { -brand-name-firefox } on your devices is completely safe — but always make sure you are downloading from a trusted { -brand-name-mozilla }/{ -brand-name-firefox } site, <a href="{ $url }">like our download page</a>.
+
+is-firefox-safe = Is { -brand-name-firefox } safe?
+
+# Variables:
+# $url (url) - link to https://www.mozilla.org/firefox/privacy/products/
+
+not-only-is = Not only is { -brand-name-firefox } safe to use, it also helps keep your data and private information safe. The { -brand-name-firefox-browser } automatically blocks known third party trackers, social media trackers, cryptominers and fingerprinters from collecting your data. Learn more about the <a href="{ $url }">privacy in our products</a>.
```

Comment:
Copy (nit): The link should be "Learn more about the privacy in our products."

<img width="652" alt="image" src="https://user-images.githubusercontent.com/2692333/96772846-561bdf00-13a9-11eb-9f19-cf07d441f35e.png">

<img width="650" alt="image" src="https://user-images.githubusercontent.com/2692333/96772849-574d0c00-13a9-11eb-9fa1-569a0f6fbbbf.png">